config/jobs/kubernetes/wg-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -167,14 +179,22 @@ periodics:
         repo:kubernetes/kubernetes
         repo:kubernetes/test-infra
       - --token=/etc/github-token/token
+      - --endpoint=http://ghproxy.default.svc.cluster.local
       - |-
-        --comment=/retest
-        This bot automatically retries jobs that failed/flaked on approved PRs (send feedback to [fejta](https://github.com/fejta)).
+        --comment=The kubernetes project has merge-blocking tests that are too flaky to consistently pass.
```

Comment:
I feel like we should add some text about how to help us improve this situation if you're so inclined.

Reply:
If anyone has any suggestions I'll incorporate into this commit, otherwise I would prefer we save for a followup

Reply:
Maybe we can reword it something like `The present number of contributors for Kubernetes project are not enough for adequately responding to all issues and PRs`. Saying the same thing passively makes it sound less sad to me : )

Reply:
ref: https://github.com/kubernetes/test-infra/pull/23094#discussion_r682330957 I was thinking the deflaking guide perhaps, or maybe pointing to #sig-testing or something.